pages/common/gnuplot.md

```diff
@@ -0,0 +1,19 @@
+# gnuplot
+
+> Optionally interactive graph plotter that outputs in several formats.
```

Comment:
What does "Optionally" mean here ? It's not very clear.

Reply:
Basically you can either interact with it via a sort of shell, or put the same commands in a file and tell it to parse it (which is what's done most of the time).

Reply:
Got it. I think it's an implementation detail which need not be mentioned in the heading. "A graph plotter that ..." sounds just fine to me.